the-telephone-app-fe/src/components/ShowResult/ShowResult.js

```diff
@@ -60,84 +63,108 @@ const ShowResult = () => {
   };
 
   return (
-    <div className="sr-screen">
-      <div className="sr-content">
-        <div className="sr-header">
-          <button className="sr-btn-back" onClick={handleButtonHome}>
-            <BsFillCaretLeftFill />
-            HOME
-          </button>
-          <img src={imgLogo} alt="logo" className="sr-img-logo" />
-          <button className="sr-btn-sound">
-            <BsFillVolumeUpFill size={"24px"} />
-          </button>
-        </div>
-        <div className="sr-sub-header">
-          <div className="sr-sub-left">PLAYERS</div>
-          <div className="sr-sub-right">
-            {location.state?.data[player].nickname}'S ALBUM
+    <div className="all">
+      <div className="main">
+        <div className="row h-20">
+          <div className="col-2 center y">
+            <button className="button">
+              <BsFillCaretLeftFill className="icon" />
+              BACK
+            </button>
+          </div>
+          <div className="col-8 center y">
+            <img src={imgLogo} alt="" className="img-logo" />
           </div>
+          <div className="col-2"></div>
         </div>
-        <div className="sr-center">
-          <div className="sr-left">
-            {users.map((user) => (
-              <div className="sr-player">
-                <img className="sr-img-avatar" src={imgAvatar} alt="avatar" />
-                <span className="sr-name">{user.nickname}</span>
+        <div className="row h-80">
+          <div className="col-4 flex-column section">
+            <div className="row h-13 y center text-title">PLAYERS ?/?</div>
+            <div className="row h-2"></div>
+            <div className="row h-80 px-2 ">
+              <div className=" scrollable">
+                {users.map((user) => (
+                  <div className="row h-20 tag-name">
```

Comment:
Add key to each array item
https://legacy.reactjs.org/docs/lists-and-keys.html#keys